be/src/agent/agent_server.cpp

```diff
@@ -208,17 +208,14 @@ void AgentServer::submit_tasks(TAgentResult& agent_result,
 
 void AgentServer::make_snapshot(TAgentResult& t_agent_result,
                                 const TSnapshotRequest& snapshot_request) {
-    Status ret_st;
     string snapshot_path;
     bool allow_incremental_clone = false;
-    OLAPStatus err_code =
+    Status err_code =
             SnapshotManager::instance()->make_snapshot(snapshot_request, &snapshot_path, &allow_incremental_clone);
-    if (err_code != OLAP_SUCCESS) {
+    if (!err_code) {
```

Comment:
why not err_code.ok()? !err_code is same with !err_code.ok() ??

Reply:
I think maybe ret_code is better.
```err_code ``` is not a good name.It seems like ```!error_code``` means no error.

Reply:
I think we should not modify variable name in this PR. Because there maybe multiple status variable in same function. Modify the variable name may cause logic bug. And also, there are too many variables, it may take lot of time.
We may refactor the status variable name in other PRs.